### DISCUSSION TITLE
Winget Software Installations via GPO/PS without Intune/SCCM

### DISCUSSION THREAD

**Discussion by workpush** [Romanitho/Winget-Install#102] [Q&A]
Hello everyone,

I want to use Winget to deploy software within our domain to our desktop clients. Our users do not have admin rights. For that I only have GPO / powershell available.
Does anyone have a strategies or give me an example configuration what is the best way to achieve that?

My naive plan would be to:

1. create a powershell script with the content:
```ps
powershell.exe -ExecutionPolicy bypass -File winget-install.ps1 -AppIDs Notepad++.Notepad++
```
and save that to \\domain.lan\SYSVOL\domain.lan\scripts\deploy\deployapps.ps1

2. Download the latest release(1.11.3) from: https://github.com/Romanitho/Winget-Install/releases
and extract the winget-install.ps1, etc directly in the directory \\domain.lan\SYSVOL\domain.lan\scripts\deploy\

3. Create a GPO:
Computer Policies->Windows Settings->Scripts->Startup->Powershell and put \\domain.lan\SYSVOL\domain.lan\scripts\deploy\deployapps.ps1 there

Is there a more intelligent way? Specific configuration examples would help me a lot, but I'm also very thankful for a hint.
When I search the internet I find literally everything except what would be the best solution for that. :/ Even my Google Foo is usually not the worst...

Thank you very much!

**Comment by Romanitho** [comment]
Hi,
I don't know what is the best approach with GPO only. I always use SCCM or Intune to deploy my apps.
You can try your plan and see what happen :)

**Comment by Dylan96** [comment]
@workpush Have you ever been able to make it work? I'm trying to deploy winget apps with gpo as well....

**Reply by aranedacristobal** [reply]
I´ve had as long as winget is available in the machine you can install whatever app. As long as you do it in the user context: ex
winget install "azure vpn" --accept-package-agreements --accept-source-agreements 

What I am having trouble with is installing winget to a machine without admin-rights
